openedx/core/djangoapps/credentials/utils.py

```diff
@@ -69,7 +79,7 @@ def get_user_program_credentials(user):
         log.debug('Display of certificates for programs is disabled.')
         return programs_credentials_data
 
-    credentials = get_user_credentials(user)
+    credentials = get_credentials(user)
```

Comment:
@mikedikan because the "user_credentials" API no longer exists. It was replaced by the more concisely named "credentials" API. The name of this function was never updated. "Always leave the campground cleaner than you found it."